tests/queries/0_stateless/02998_native_parquet_reader.sh

```diff
@@ -0,0 +1,211 @@
+#!/usr/bin/env bash
+# Tags: no-fasttest
+
+CURDIR=$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)
+# shellcheck source=../shell_config.sh
+. "$CURDIR"/../shell_config.sh
+
+PAR_PATH="$CURDIR"/02998_native_parquet_reader.parquet
+# the content of parquet file can be generated by following codes
```

Comment:
The dictionary page size can not be configured by the parquet writer of ClickHouse, which is very important to test different logic in native parquet reader. E.g. some columns may be written totally in dictionary format; but some columns may be written partially in dictionary format, then changed to normal data page. So I have to generate the corresponding test cases with a parquet file as smaller as better.

Further more, I think the native reader should support all test cases in 02735, so I prefer to enable the whole test case when it's possible, instead of a partial copy of it temporarily. And I also will work on this feature when this PR is finished.